### PR TITLE
swimming

### PR DESCRIPTION
**Pull request by ayrka39**

### CODE REVIEW

**Comment by ayrka39** [comment]
Please review this, @casey-collab

File: english/swimming/butterfly.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+full of mashmallow.
```
Comment (casey-collab) [inline english/swimming/butterfly.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

File: swimming/butterfly.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Full of mashmallow.
```
Comment (casey-collab) [inline swimming/butterfly.md]: This page is also unreachable.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Pull request closed without merging**